x-pack/plugins/ml/server/routes/trained_models.ts

```diff
@@ -237,6 +239,112 @@ export function trainedModelsRoutes({ router, routeGuard }: RouteInitialization)
       })
     );
 
+  /**
+   * @apiGroup TrainedModels
+   *
+   * @api {get} /internal/ml/trained_models/ingest_pipelines Get ingest pipelines
+   * @apiName GetIngestPipelines
```

Comment:
This API needs to be added to the config for the docs script - https://github.com/elastic/kibana/blob/main/x-pack/plugins/ml/scripts/apidoc_scripts/apidoc_config/apidoc.json